hphp/runtime/vm/jit/vasm-ppc64.cpp

```diff
@@ -707,6 +712,14 @@ void Vgen::emit(const mcprep& i) {
 }
 
 void Vgen::emit(const inittc&) {
```

Comment:
I'm pretty sure you guys added `inittc{}`, so its only real purpose is to allow the PPC64 backend to do additional work before anything else happens in enterTCHelper.  You can do whatever you want in it :]